flow/scripts/cts.tcl

```diff
@@ -73,6 +73,11 @@ if {[info exist ::env(CTS_SNAPSHOTS)]} {
   save_progress 4_1_pre_repair_hold_setup
 }
 
+write_timing_model latency.lib
+# FIXME! How do I find the average clock insertion latency?
+set max_latency [exec bash -c {python -c "import re; print(re.search('max_clock_tree_path.*?cell_fall\(scalar\).*?values\(\"(.*?)\"\)', open('latency.lib').read(), re.DOTALL).group(1))"}]
+set_clock_latency [expr $max_latency] [get_clocks {clock_vir}]
```

Comment:
What is the purpose of this?  You have a clock tree so why set clock latency at all when you can use propagated clocks?

Reply:
Otherwise you get a ton of hold cells.

Reply:
recall: Louii and My adviced this approach last summer and you/tom approved...

Reply:
The propagated clocks work at the top level but when scoped on the block itself the top level latency is pushed into the blocks with set_clock_latency.